File: VB/B3013/Program.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.Skins
Imports System.Windows.Forms

Namespace B3013
	Friend NotInheritable Class Program
		''' <summary>
		''' The main entry point for the application.
		''' </summary>
		Private Sub New()
		End Sub
		<STAThread> _
		Shared Sub Main()
			Application.EnableVisualStyles()
			Application.SetCompatibleTextRenderingDefault(False)
			SkinManager.EnableFormSkins()
			Application.Run(New MainForm())
		End Sub
	End Class
End Namespace

File: CS/B3013/Program.cs
﻿using System;
using DevExpress.Skins;
using System.Windows.Forms;

namespace B3013 {
    static class Program {
        /// <summary>
        /// The main entry point for the application.
        /// </summary>
        [STAThread]
        static void Main() {
            Application.EnableVisualStyles();
            Application.SetCompatibleTextRenderingDefault(false);
            SkinManager.EnableFormSkins();
            Application.Run(new MainForm());
        }
    }
}